client/state/jetpack-connection-health/selectors/is-requesting-jetpack-connection-health-status.js

```diff
@@ -0,0 +1,16 @@
+import 'calypso/state/jetpack-connection-health/init';
+
+/**
+ * Returns true if we are currently making a request to retrieve the connection status. False otherwise.
+ * Returns null if the site is unknown, or there is no information yet.
```

Comment:
This is currently incorrect; it will return `undefined` if there is no information about that site.